specs/Print.md

```diff
@@ -0,0 +1,861 @@
+  COREWEBVIEW2_PRINT_COLLATION_UNCOLLATED,
+
+} COREWEBVIEW2_PRINT_COLLATION;
+
+/// Specifies the media size for a print.
+[v1_enum] typedef enum COREWEBVIEW2_PRINT_MEDIA_SIZE {
+  /// The default media size for a printer.
+  COREWEBVIEW2_PRINT_MEDIA_SIZE_DEFAULT,
+
+  /// Indicate custom media size that is specific to the printer.
+  COREWEBVIEW2_PRINT_MEDIA_SIZE_CUSTOM,
+
+} COREWEBVIEW2_PRINT_MEDIA_SIZE;
+
+[uuid(92269C88-1CC1-47E0-9A87-71458409BA15), object, pointer_default(unique)]
+interface ICoreWebView2_15 : ICoreWebView2_14 {
+  /// Print the current web page asynchronously to the specified printer with the provided settings.
+  /// See `ICoreWebView2PrintSettings2` for description of settings.
+  /// The handler will return `errorCode` as `S_OK` and `isSuccessful` as FALSE if `printerName`
+  /// doesn't match with the name of any installed printers on the user OS.
+  /// The handler will return `errorCode` as `E_INVALIDARG` and `isSuccessful` as FALSE
+  /// if the caller provides invalid settings for a given printer.
+  ///
+  /// The async `Print` operation completes when it finishes printing to the printer.
+  /// At this time the `ICoreWebView2PrintCompletedHandler` is invoked. Only one `Printing` operation
+  /// can be in progress at a time. If `Print` is called while a `Print` or `PrintToPdf`
+  /// or `PrintToPdfStream` or `ShowPrintUI` job is in progress, the completed handler is
+  /// immediately invoked with `E_ABORT` and `isSuccessful` set to FALSE.
+  ///
+  /// \snippet AppWindow.cpp PrintToPrinter
+  HRESULT Print(
+    [in] ICoreWebView2PrintSettings2* printSettings,
+    [in] ICoreWebView2PrintCompletedHandler* handler);
+
+  /// Opens the print dialog to print the current web page. See `COREWEBVIEW2_PRINT_DIALOG_KIND`
+  /// for descriptions of print dialog kinds.
+  ///
+  /// \snippet AppWindow::ShowPrintUI
+  HRESULT ShowPrintUI([in] COREWEBVIEW2_PRINT_DIALOG_KIND printDialogKind);
+
+  /// Provides the Pdf data of current web page asynchronously for the provided settings.
+  /// See `ICoreWebView2PrintSettings2` for description of settings. Passing
+  /// nullptr for `printSettings` results in default print settings used.
+  ///
+  /// The async `PrintToPdfStream` operation completes when it finishes
+  /// writing to the stream. At this time the `ICoreWebView2PrintToPdfStreamCompletedHandler`
+  /// is invoked. Only one `Printing` operation can be in progress at a time. If
+  /// `PrintToPdfStream` is called while a `PrintToPdfStream` or `PrintToPdf` or `Print` or `ShowPrintUI`
+  /// job is in progress, the completed handler is immediately invoked with `E_ABORT`.
+  ///
+  /// \snippet AppWindow.cpp PrintToPdfStream
+  HRESULT PrintToPdfStream([in] ICoreWebView2PrintSettings2* printSettings,
+                           [in] ICoreWebView2PrintToPdfStreamCompletedHandler* handler);
+}
+
+/// Receives the result of the `Print` method.
```

Comment:
This info is good! Please put it with the Print method rather than the completedhandler interface.